### ISSUE TITLE
How can I upgrade v1.8 to v1.10

### ISSUE THREAD

**Issue by christhomas** [kubernetes/kubeadm#888]
I've got a cluster which is on 1.8, I'm trying to upgrade so I'm trying to do a 'kubeadm upgrade plan' and I'm told that:

```
[upgrade/config] FATAL: could not decode configuration: this version of kubeadm only supports deploying clusters with the control plane version >= 1.9.0. Current version: v1.8.13
```

I followed the instructions on this page: https://kubernetes.io/docs/tasks/administer-cluster/upgrade-downgrade/kubeadm-upgrade-1-9/

and I downloaded the kubeadm file as given in those instructions. 

So can anybody tell me how I can move forward?

**Comment by luxas** [comment]
You need to upgrade v1.8 to v1.9 first, and then v1.9 to v1.10.
Cheers!

**Comment by christhomas** [comment]
I'm really unsure why you closed this issue without further comment from me, and it appears clear that you did not read what I wrote. So how do you consider this issue closed?

**Comment by luxas** [comment]
Sorry 2am in the evening and misread the comment indeed. Download kubeadm v1.9.x instead, using `https://dl.k8s.io/release/stable-1.9.txt)`. Does that solve it for you?

**Comment by christhomas** [comment]
I'm going to try it out and see. There perhaps should be updated documentation to put the correct link into the instructions?

**Comment by christhomas** [comment]
Sorry, I forgot to follow up on this. The reason that I could not upgrade was because of the hostname of the machine.

If I set the hostname to s1.antimatter-studios.com. It would fail to upgrade. If I simply changed the hostname to s1. Then it would upgrade without any problem.

I could constantly set, test, set, retest. And the problem was reproducible and repeatable with perfect accuracy.

So it seems like the hostname of the machine is very important.
